lang/zh.json

```diff
@@ -427,9 +427,9 @@
         "transferPartyOwner": "{PLAYER} 已被转授为领队。"
       },
       "events": {
-        "complete": "成功抵达 {LOCATION} 。远征告捷！您已获得 {EXP} 。",
+        "complete": "成功抵达 {LOCATION} 。\n远征告捷！您已获得 {EXP} 。",
         "freeComplete": "顺利造访了 {LOCATION} 。狂梦未散，流浪不止！",
```

Comment:
Why no new line for this one?

Reply:
Second half is supposed to be in a new line.

Reply:
The original english message wasn't splitted into two lines so I didn't.

Reply:
Oh, my bad